spec/operations/story_operations/destroy_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe StoryOperations::Destroy do
+  let(:story_params) do
+    { title: 'Test Story', requested_by: user, state: 'unstarted', accepted_at: nil }
+  end
+
+  let!(:membership) { create(:membership) }
+  let(:user)        { membership.user }
+  let(:project)     { membership.project }
+  let(:story)       { project.stories.build(story_params) }
+
+  before { story.save! }
```

Comment:
why not create a new story directly on line 11 so that you don't need to worry about having to save it on the before block? Also if you're in fact destroying something, it would be good to have a test for this with `change count by (-1)` for instance.